pkg/docker/docker.go

```diff
@@ -18,41 +18,91 @@ import (
 	docker "github.com/docker/docker/client"
 	"github.com/docker/docker/pkg/archive"
 	"github.com/kyma-project/cli/internal/minikube"
+	"github.com/kyma-project/cli/pkg/step"
 )
 
 const (
 	defaultRegistry = "index.docker.io"
 )
 
-// DockerErrorMessage is used to parse error messages coming from Docker
-type DockerErrorMessage struct {
+type dockerClient struct {
+	*docker.Client
+	// DockerClientService
+}
+
+type kymaDockerClient struct {
+	kymaDocker ClientService
+}
+
+//go:generate mockery --name ClientService
+type ClientService interface {
```

Comment:
This interface name is too generic. I would suggest something like `DockerClient`.

Reply:
I tried DockerClient but golint say people would stutter while reading it and suggested to change it to ClientService.

Reply:
Of course, because of being in the context of the docker package. In that case, the interface would be just `Client`.
Same for the `KymaDockerService`.

I would also drop the `Service` suffix alltogether since it does not have any meaning.

Reply:
fixed.